foundation-extensions/lifecycle-for-edge-network/README.md

```diff
@@ -1,42 +1,42 @@
+The Adobe Experience Platform Mobile SDK Lifecycle extension enables application lifecycle data collection from your mobile app when using the Adobe Experience Platform Mobile SDK and the [Edge Network extension](../experience-platform-extension/). Use the following steps to setup an application to forward Lifecycle extension metrics to the Adobe Experience Platform.
 
 
 ## Add AEP Mobile Lifecycle Details field group to your XDM Schema in Adobe Experience Platform
 
-Include the AEP Mobile Lifecycle Details field group to the schema defined in your Edge configuration data stream. The AEP Mobile Lifecycle field group defines all the metrics auto-collected by the Lifecycle for Edge Network extension. 
+Include the "AEP Mobile Lifecycle Details" field group to the schema defined in your Edge configuration data stream. The "AEP Mobile Lifecycle Details" field group defines all the metrics auto-collected by the Lifecycle for Edge Network extension. 
 
-1. In Adobe Experience Platform, log into your organization.
+1. In Adobe Experience Platform, log in to your organization.
 2. Under **Schemas** section, select the **Browse** tab and search for the schema used in the application's Edge configuration.
-3. In the **Field groups** section, select **Add**
-4. Search for **AEP Mobile Lifecycle Details**, select it then click **Add field groups**
-5. Click **Save**
+3. In the **Field groups** section, select **Add**.
+4. Search for **AEP Mobile Lifecycle Details**. Select it, then click **Add field groups**.
+5. Click **Save**.
 
 ![](../../.gitbook/assets/xdm-schema-add-mobile-lifecycle-details.png)
 
 
-## Configure the Adobe Experience Platform Lifecycle extension in Experience Platform Launch
+## Configure the Adobe Experience Platform Lifecycle extension in Experience Platform
 
 {% hint style="info" %}
-The Lifecycle for Edge Network extension is part of the Mobile Core extension. There is no separate card for the Lifecycle for Edge Network extension in the Launch Extensions Catalog.
+The Lifecycle for Edge Network extension is part of the Mobile Core extension. There is no separate card for the Lifecycle for Edge Network extension in the Extensions Catalog.
 {% endhint %}
 
 {% hint style="warning" %}
 The configuration setting "Session Timeout" in the Mobile Core extension configuration is used when collecting Lifecycle session data for Analytics and is not used for the Lifecycle for Edge Network extension.
 {% endhint %}
 
-1. In Experience Platform Launch, in your mobile property, click the **Extensions** tab.
+1. In Experience Platform, in your tags mobile property, click the **Extensions** tab.
```

Comment:
I would suggest to use "mobile property" instead of "tags mobile property" for all occurrences.

Reply:
In Platform, it is named "Tags" and "tag properties". Do you know if this will change in the future?  I would prefer to use the same language used in Platform.